packages/tables/resources/lang/tr/table.php

```diff
@@ -59,6 +59,31 @@
 
     ],
 
+
+    'bulk_actions' => [
```

Comment:
These translations are outdated, they have moved to support/actions

Reply:
They're also still in the `en` translations file. Should they be removed there as well, @danharrin?

Reply:
oh, really? that must be a merge problem then, yeah they need removing everywhere

Reply:
https://github.com/filamentphp/filament/pull/2886 @danharrin